src/installer-source/KolibriSetupScript.iss

```diff
@@ -645,6 +645,36 @@ begin
         
         EnvAddPath(PipDir) 
 
+        { Delete existing system and user KOLIBRI_INSTALLATION_TYPE environment variables }
+        RegDeleteValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'KOLIBRI_INSTALLATION_TYPE'
+        )
+        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_INSTALLATION_TYPE"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)
+
+        { Delete existing system and user KOLIBRI_INSTALLER_VERSION environment variables }
+        RegDeleteValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'KOLIBRI_INSTALLER_VERSION'
+        )
+        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_INSTALLER_VERSION"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)
+
+        RegWriteStringValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'KOLIBRI_INSTALLATION_TYPE',
+            'windows'
+        );
+
+        RegWriteStringValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'KOLIBRI_INSTALLER_VERSION',
+            'v1.5.0'
+        );
```

Comment:
main thing I'd be interested in making sure of is that these variables are accessible after installation, not just during the setup script run

Reply:
The other env vars referenced in here are used at run time, so my assumption was yes.

Reply:
See here for example: https://github.com/learningequality/kolibri-installer-windows/blob/a8ea0684b38131bc60cacd5cbfb8830bd2c8a2ff/src/gui-source/Kolibri/Kolibri.cpp#L99